src/losc/init.lua

```diff
@@ -158,6 +158,13 @@ function losc:open(...)
   return pcall(self.plugin.open, self.plugin, ...)
 end
 
+function losc:poll(...)
```

Comment:
you beat me to the PR!
I would add some documentation here:
```
--- Polls an OSC server.
-- Used by non-blocking plugins.
-- @param[opt] ... Plugin specific arguments.
-- @return bool representing if the plugin dispatched a message
-- @usage losc:poll()
```

Reply:
Thanks I'll add it!
I should maybe link to the PIL page that has an example of this as well: https://www.lua.org/pil/9.4.html
It's about coroutines but the purpose of the :timeout() method is clearly shown